src/WebJobs.Script.WebHost/App_Data/secrets/host.json

```diff
@@ -1,4 +1,4 @@
 ﻿{
-  "masterKey": "t8laajal0a1ajkgzoqlfv5gxr4ebhqozebw4qzdy",
-  "functionKey": "zlnu496ve212kk1p84ncrtdvmtpembduqp25ajjc"
+    "masterKey": "t8laajal0a1ajkgzoqlfv5gxr4ebhqozebw4qzdy,z3dlq50s00cb3q3k11nil7xyt29ebst2n8rtn0t3",
```

Comment:
Clearly both David and I read #270 to mean something different. This came out of security review, and I think we both interpreted it as a need to roll keys. We need to clarify what #270 was actually for - I'll do that.